mailbox/lucene/src/main/resources/META-INF/spring/mailbox-index-lucene.xml

```diff
@@ -31,10 +31,10 @@
         <constructor-arg index="0" ref="luceneIndex"/>
     </bean>
     <bean id="luceneIndex" class="org.apache.james.mailbox.lucene.search.LuceneMessageSearchIndex">
-        <constructor-arg index="0" ref="maildir-sessionMapperFactory"/>
+        <constructor-arg index="0" ref="jpa-sessionMapperFactory"/>

```

Comment:
"*-sessionMapperFactory" are still there for purpose of multiple stores. The "spring-mailbox.xml" includes some of them. (memory, cassandra, jpa ...)
We can bind only sessionMapperFactory but we have to add more alias at another place and I don't think we would like to make that more complicated. WDYT?